vllm/platforms/rocm.py

```diff
@@ -164,6 +164,7 @@ class RocmPlatform(Platform):
     device_type: str = "cuda"
     dispatch_key: str = "CUDA"
     ray_device_key: str = "GPU"
+    dist_backend: str = "gloo"
```

Comment:
```suggestion
    dist_backend: str = "nccl"
```
I think ROCm also use `nccl` as distributed backend.